Frontend/src/lib/forms/AddUserForm.svelte

```diff
@@ -19,53 +19,83 @@
     let group = [];
 </script>
 
+<div>
+    <h2>
+        Create a User
+        <hr />
+    </h2>
+    
+    <form action="" method="POST" on:submit|preventDefault={handleSubmit} class="add-form" ><!--  prevent default is used to stop the page from refresh -->
+        
 
-<form action="" method="POST" on:submit|preventDefault={handleSubmit} class="add-form" ><!--  prevent default is used to stop the page from refresh -->
-    <label>
-        Title
-        <input type="text" placeholder="" bind:value={title}>
-    </label>
-
-    <label>
-        First Name
+        <!-- Revisit labels, it seemed at default the label tags do not work with the required-field class -->
```

Comment:
Probably not the best way to remind yourself to do something. Maybe add a task on an appropriate story or create one. High likelyhood unless this is feature blocking you will forget

Reply:
I will delete that note as we discussed it in a meeting.